evalml/tests/model_understanding_tests/test_permutation_importance.py

```diff
@@ -398,10 +398,25 @@ def test_fast_permutation_importance_matches_slow_output(
         )
 
 
+class MockFeature:
+    def __init__(self, name) -> None:
+        self.name = name
+
+    def get_feature_names(self):
+        return self.name
+
+
 def pipelines_that_do_not_support_fast_permutation_importance():
-    pipeline_with_dfs = BinaryClassificationPipeline(
+    pipeline_with_dfs_and_no_prexisting_features = BinaryClassificationPipeline(
```

Comment:
Can you add the opposite case to `test_fast_permutation_importance_matches_slow_output` so that we have coverage for DFS with prexisting features?

Reply:
done